cmake/macros/rv_create_std_deps_vars.cmake

```diff
@@ -0,0 +1,104 @@
+#
```

Comment:
We need to add the standard header otherwise the ASWF license scanner will complain:
#
# Copyright (C) 2023  Autodesk, Inc. All Rights Reserved.
#
# SPDX-License-Identifier: Apache-2.0
#